server/_tests_/api.spec.js

```diff
@@ -0,0 +1,2 @@
+const request = require('supertest');
+const app = require('../app');
```

Comment:
As it is right now, this file is not necessary. If you don't end up adding anything else to it you can delete it